objects/type_string.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by "stringer -type Type"; DO NOT EDIT.
 
-package objects
+package objects // import "gosh-lang.org/gosh/objects"
```

Comment:
That's a generated file. We should check that generated files are up-to-date on CI.